Sources/CasePaths/Internal/Deprecations.swift

```diff
@@ -227,7 +227,7 @@ public func XCTUnwrap<Enum, Case>(
         Actual:
           \(String(describing: `enum`))
       """,
-      file: file,
+      file: (file),
```

Comment:
I think we want to silence this warning right?

> ⚠️ Parameter 'file' with default argument '#file' passed to parameter 'file', whose default argument is '#filePath'

Reply:
I _think_ the way to silence this is to stop using `#file` since its behavior changes in Swift 6 language mode and instead use `#filePath explicitly.